internal/provider/provider.go

```diff
@@ -171,10 +175,9 @@ func getRetryConfig(ctx context.Context, d *schema.ResourceData) (kentikapi.Retr
 	// KentikAPI returns http 500 when creating CloudExport with name that is already taken.
 	// This is non-retryable situation, so exclude http 500 from RetryableStatusCodes
 	return kentikapi.RetryConfig{
-		MaxAttempts:          &maxAttempts,
-		MinDelay:             &minDelay,
-		MaxDelay:             &maxDelay,
-		RetryableStatusCodes: []int{429, 502, 503, 504},
+		MaxAttempts: &maxAttempts,
+		MinDelay:    &minDelay,
+		MaxDelay:    &maxDelay,
```

Comment:
Does it mean that every unsuccessful operation will be retried regardless of the type of error? AFAIk, gRPC all has response error codes.

Reply:
makeRetryInterceptor() determines on which error code make retries. Only in this interceptor we can configure it.
https://github.com/kentik/community_sdk_golang/blob/2fb7ec871b8d1e6bdcde6f5e86defb49dda40237/kentikapi/client.go#L268

Reply:
To make it clear: we have removed an option to configure retryable status code in Kentik Go client to simplify the code. @jdziurzynski pointed to the code responsible for configuring gRPC retries in the client.

Reply:
OK. We should retry any transient errors, i.e. RESOURCE_EXHAUSTED (8), FAILED_PRECONDITION (9), ABORTED (10), UNAVAILABLE (14).

Reply:
Ticket created: https://clkentik.atlassian.net/browse/KNTK-404